packages/zero-cache/src/server/main.ts

```diff
@@ -147,6 +147,10 @@ export default async function runWorker(
     }
     syncers.forEach(syncer => handleSubscriptionsFrom(lc, syncer, notifier));
   }
+  let mutator: Worker | undefined;
+  if (config.push.url !== undefined) {
```

Comment:
custom mutators are optional until we are ready to release them and deprecate crud mutators.